libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -73,45 +123,42 @@ class NavigationView : ConstraintLayout, LifecycleObserver {
         NavigationViewOptions.Builder().build()
     )
 
-    @JvmOverloads
     constructor(
         context: Context,
-        attrs: AttributeSet? = null,
+        attrs: AttributeSet?,
         accessToken: String?,
-        mapInitOptions: MapInitOptions = MapInitOptions(context),
-        navigationViewOptions: NavigationViewOptions = NavigationViewOptions.Builder().build()
+        mapInitializationOptions: MapInitOptions,
+        navigationViewOptions: NavigationViewOptions
```

Comment:
As I recall there was some kind of conflict the compiler was complaining about. Ultimately everything is funneled through the final constructor which does the assignment.  I don't remember the exact details.

Reply:
I would encourage to add these default values back as I am able to do it in your branch locally and its compiling & running for me. This would make it even easier for developers who just want to use it out of the box without being bothered by the need to create options.

Reply:
There's a constructor that takes only a context and attributeSet. That constructor will create default values for the options.

Reply:
Now, there's no constructor that let's you pass just `context` and `token`. It'd be good to add the defaults back.